TouristEquipmentRental/UserPanel.cpp

```diff
@@ -1,116 +1,197 @@
+
+    switch (choice)
+    {
+    case '1':
+        ShowUserData(user);
+        RenderNavigationBar(user, 50);
+        break;
+
+    case '2':
+        ShowUserReservations(user);
+        RenderNavigationBar(user, 110);
+        break;
+    case '3':
+        RentEquipmentFlow(user);
+        RenderNavigationBar(user, 110);
+        break;
+    case '4':
+        exit(0);
+        break;
+
+    default: cout << "Nie ma takiej opcji w menu!";
+    };
+
+
+}
 void UserPanel::ShowUserData(User user) {
 
     system("cls");
 
-    int  primmaryC = 11,
-         secconadryC = 14,
-         fieldLentgth = 30;
+    int fieldLentgth = 30;
 
-    /*separator*/
     style.SetColor(102);
     style.CreateSeparator(50, ' ');
-    style.SetColor(primmaryC);
+    style.SetColor(11);
 
-    /*Konto*/
     cout << endl;
-    cout <<setfill(' ') << setw(15) << "KONTO:" << setw(fieldLentgth);
-    style.SetColor(secconadryC);
+    cout << setfill(' ') << setw(15) << "RODZAJ KONTA:" << setw(fieldLentgth);
+    style.SetColor(14);
     (user.Id()[0] == 'A') ? cout << "ADMINISTRATOR" << endl : cout << "UZYTKOWNIK" << endl << endl;
 
-    /*Status konta*/
-    style.SetColor(primmaryC);
-    cout << setw(15) << "STATUS KONTA:" << setw(fieldLentgth); 
-    style.SetColor(secconadryC);
+    style.SetColor(11);
+    cout << setw(15) << "STATUS KONTA:" << setw(fieldLentgth);
+    style.SetColor(14);
+
     if (user.Activated() == true) {
         style.SetColor(2);
-        cout << "AKTYWOWANY" << endl;
+        cout << "AKTYWOWANY\n" << endl;
     }
     else {
         style.SetColor(4);
-        cout << "NIEAKTYWNY" << endl;
+        cout << "NIEAKTYWNY\n" << endl;
     }
-    cout << endl;
 
-    /*Imie*/
-    style.SetColor(primmaryC);
-    cout << setw(15) << "IMIE:" << setw(fieldLentgth);  
-    style.SetColor(secconadryC);
-    cout<< user.FirstName() << endl << endl;            
-    style.SetColor(primmaryC);
+    style.SetColor(11);
+    cout << setw(15) << "IMIE:" << setw(fieldLentgth);
+    style.SetColor(14);
+    cout << user.FirstName() << endl << endl;
 
-    /*Nazwisko*/
+    style.SetColor(11);
     cout << setw(15) << "NAZWISKO:" << setw(fieldLentgth);
-    style.SetColor(secconadryC);
+    style.SetColor(14);
     cout << user.LastName() << endl << endl;
-    style.SetColor(primmaryC);
 
-    /*Email*/
-    cout << setw(15) << "EMAIL:" << setw(fieldLentgth); 
-    style.SetColor(secconadryC);
-    cout << user.Email() << endl << endl;                
+    style.SetColor(11);
+    cout << setw(15) << "EMAIL:" << setw(fieldLentgth);
+    style.SetColor(14);
+    cout << user.Email() << endl << endl;
     cout << endl;
 
-    /*separator*/
     style.SetColor(102);
     style.CreateSeparator(50, ' ');
-    style.SetColor(primmaryC);
+    style.SetColor(14);
+
+}
+
+
+void UserPanel::ShowUserReservations(User user) {
 
+    system("cls");
+
+    auto fileDatabase = new FileDatabase();
+    auto databaseOperator = new DatabaseOperator(*fileDatabase);
```

Comment:
Each view should contain FileDatabase reference or reference of DatabaseOperator, we shouldn't create more than one instance of them.